arcgisUM.py

```diff
@@ -102,8 +97,8 @@ def addCanvasUsersToGroup(instructorLog, group, courseUsers):
         return instructorLog
 
     logger.info('Adding Canvas Users to ArcGIS Group {}: {}'.format(groupNameAndID, courseUsers))
-    # ArcGIS usernames are U-M uniqnames with the ArcGIS organization name appended.
-    arcGISFormatUsers = formatUsersNamesForArcGIS(courseUsers)
```

Comment:
Since you've moved the use of `formatUsersNamesForArcGIS()` out of this method, note that the `courseUsers` appearing in the logging on line 99 are technically no longer "Canvas Users".  You should do one of these:

1. Change the text of the logging on line 99.
1. Return the call to `formatUsersNamesForArcGIS()` to this method.

This method was written to have its process self-contained and make calls to it simple.  By simple, I mean passing it a list of uniqnames, rather than requiring the uniqnames to be changed to ArcGIS format before the call.

Reply:
Good point.  I've removed explicit references to Canvas from the arcgisUM file.